pkg/network/vmispec/interface_test.go

```diff
@@ -88,6 +89,35 @@ var _ = Describe("VMI network spec", func() {
 			Expect(netvmispec.FilterSRIOVInterfaces(ifaces)).To(Equal([]v1.Interface{sriov_net1, sriov_net2}))
 		})
 	})
+
+	const iface1, iface2, iface3, iface4, iface5 = "iface1", "iface2", "iface3", "iface4", "iface5"
+
+	table.DescribeTable("return VMI spec interface names, given",
+		func(interfaces []v1.Interface, expectedNames []string) {
+			Expect(netvmispec.InterfacesNames(interfaces)).To(Equal(expectedNames))
+		},
+		table.Entry("no interfaces", nil, nil),
+		table.Entry("single interface", vmiSpecInterfaces(iface1), []string{iface1}),
+		table.Entry("more then one interface", vmiSpecInterfaces(iface1, iface2, iface3), []string{iface1, iface2, iface3}),
+	)
+
+	It("filter status interfaces, given 0 interfaces and 0 names", func() {
+		Expect(netvmispec.FilterStatusInterfacesByNames(nil, nil)).To(BeNil())
```

Comment:
You should check for `BeEmpty` not `nil`. If the slice is nil or empty is of no interest, golang supports this transparently.

Same for the others bellow.

Reply:
Thanks for the heads up :slightly_smiling_face: 
Done